SampleProject/VOKCoreDataManagerTests/DateFormatterTests.m

```diff
@@ -0,0 +1,27 @@
+//
+//  DateFormatterTests.m
+//  SampleProject
+//
+//  Created by Carl Hill-Popper on 4/27/16.
+//  Copyright © 2016 Vokal.
+//
+
+#import <XCTest/XCTest.h>
+
+#import <VOKManagedObjectMap.h>
+
+@interface DateFormatterTests : XCTestCase
+
+@end
+
+@implementation DateFormatterTests
+
+- (void)testDateFormattersLocaleIsPOSIX
+{
+    XCTAssertEqualObjects([VOKManagedObjectMap vok_defaultDateFormatter].locale.localeIdentifier,
+                          VOKDefaultDateFormatterLocaleIdentifier);
+    XCTAssertEqualObjects([VOKManagedObjectMap vok_dateFormatterWithoutMicroseconds].locale.localeIdentifier,
+                          VOKDefaultDateFormatterLocaleIdentifier);
+}
```

Comment:
AFAIK, you can only set the locale by passing in process arguments or choosing a locale in the Xcode scheme:
https://developer.apple.com/library/ios/documentation/MacOSX/Conceptual/BPInternational/TestingYourInternationalApp/TestingYourInternationalApp.html
http://useyourloaf.com/blog/using-launch-arguments-to-test-localizations/
Which makes testing that way a bit of a pain.